pkg/rebuild/stability/stabilizer.go

```diff
@@ -29,7 +29,7 @@ func StabilizersForTarget(t rebuild.Target) ([]archive.Stabilizer, error) {
 	switch t.Ecosystem {
 	case rebuild.Maven:
 		if format == archive.ZipFormat {
-			stabilizers = append(stabilizers, archive.StableJARBuildMetadata, archive.StableJAROrderOfAttributeValues, archive.StableGitProperties)
+			stabilizers = append(stabilizers, archive.AllJarStabilizers)
```

Comment:
nit: I think this should have the spread operator here:

`append(stabilizers, archive.AllJarStabilizers...)`

Reply:
This is a nitpick as it just makes the code clearer right? Otherwise, `go` seems to implcity append elements of array instead of appending the array (this would anyway fail type checking).

Reply:
Jeez I didn't realize this but because Stabilizer is an alias of `any`, `[]Stabilizer` will accept the slice as an element 🤦 

```
type Foo any

func main() {
	s := []Foo{0}
	fmt.Printf("%v\n", s)
	fmt.Printf("%v\n", append(s, 1))
	fmt.Printf("%v\n", append(s, []Foo{1}))
	fmt.Printf("%v\n", append(s, []Foo{1}...))
}
```

prints...

```
[0]
[0 1]
[0 [1]]
[0 1]
```

so we really can't rely on types here, apparently :(

Reply:
filed an issue to track.